source/game_sa/ClothesBuilder.cpp

```diff
@@ -129,7 +129,27 @@ void CClothesBuilder::BuildBoneIndexConversionTable(uint8* a1, RpHAnimHierarchy*
 
 // 0x5A5730
 RwTexture* CClothesBuilder::CopyTexture(RwTexture* texture) {
-    return plugin::CallAndReturn<RwTexture*, 0x5A5730, RwTexture*>(texture);
+    RwRaster* raster = RwRasterCreate(
+        texture->raster->width,
+        texture->raster->height,
+        texture->raster->depth,
+        ((texture->raster->cFormat << 8) & (rwRASTERFORMAT1555 | rwRASTERFORMAT565 | rwRASTERFORMAT4444 | rwRASTERFORMATLUM8 | rwRASTERFORMAT8888 | rwRASTERFORMAT888 | rwRASTERFORMAT16 |
+                    rwRASTERFORMAT24 | rwRASTERFORMAT32 | rwRASTERFORMAT555 | rwRASTERFORMATPAL8 | rwRASTERFORMATPAL4)) | 4
+    );
+
+    const auto lockedRasterPixelsOld = RwRasterLock(texture->raster, 0, rwRASTERLOCKREAD);
+    const auto lockedRasterPixelsNew = RwRasterLock(raster, 0, rwRASTERLOCKWRITE);
+
+    memcpy(lockedRasterPixelsNew, lockedRasterPixelsOld, texture->raster->height * texture->raster->stride);
+
+    RwRasterUnlock(texture->raster);
+    RwRasterUnlock(raster);
+
+    RwTexture* newTexture = RwTextureCreate(raster);
+    RwTextureSetFilterModeMacro(newTexture, 2);
```

Comment:
Don't use `Macro` stuff directly, use `RwTextureSetFilterMode`
```suggestion
    RwTextureSetFilterModeMacro(newTexture, rwFILTERLINEAR);
```

Reply:
rule of thumb: if some renderware func (i.e. Rw/Rs/Rp/Rx/Rt prefixed ones) is called with constant values, it's very likely an enum value